packages/docs-site/.vitepress/config.ts

```diff
@@ -221,6 +221,7 @@ export default defineConfig({
                       link: "/docs/ref/style/shapes/rectangle",
                     },
                     { text: "Text", link: "/docs/ref/style/shapes/text" },
+                    { text: "Group", link: "/docs/ref/style/shapes/group" },
```

Comment:
Awesome docs page, great work writing this up! Super small note, could you put this menu item in alphabetical order among the others? (Maybe add a comment here saying that these should stay in alphabetical order.)

Reply:
Done